tcp.go

```diff
@@ -11,19 +12,34 @@ import (
 	"github.com/prometheus/log"
 )
 
+func dialTCP(target string, ssl bool, timeout time.Duration) (net.Conn, error) {
+	dialer := &net.Dialer{Timeout: timeout}
+	if !ssl {
+		return dialer.Dial("tcp", target)
+	}
+	config := &tls.Config{InsecureSkipVerify: true}
```

Comment:
It should all be in the config file.

There's no mention of GET in that PR, the only things that should be passed in are module name and target.